controllers/machinedeletionremediation_controller.go

```diff
@@ -22,14 +22,16 @@ import (
 	"strings"
 
 	"github.com/go-logr/logr"
-	"github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
+
 	v1 "k8s.io/api/core/v1"
```

Comment:
Nit: i think v1 is redundant (i.e replace `v1 "k8s.io/api/core/v1"` with `"k8s.io/api/core/v1"`)